.env.example

```diff
@@ -10,11 +10,11 @@ ALCHEMY_POLYGON_API_KEY=
 ALCHEMY_ARBITRUM_API_KEY=
 IPFS_PROJECT_ID=
 IPFS_API_KEY_SECRET=
-OPT_ETHERSCAN_API_KEY=
-MAIN_ETHERSCAN_API_KEY=
-POLYGONSCAN_API_KEY=
-ARBISCAN_API_KEY=
-BASESCAN_API_KEY=
+OPT_ETHERSCAN_API_KEY=""
+MAIN_ETHERSCAN_API_KEY=""
+POLYGONSCAN_API_KEY=""
+ARBISCAN_API_KEY=""
+BASESCAN_API_KEY=""
```

Comment:
Added filler values for the Etherscan API keys because Foundry throws an internal error if they're not defined when running a Forge script.